week2/homework/index.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const fs = require('fs');
+
+let fileName = 'toDo.txt';
+// console.log(fileName);
+function readFile() {
+    return new Promise(
+        resolve => fs.readFile(
+            fileName,
+            (err, data) => resolve(err ? '' : data.toString())
+        )
+    );
+}
+
+function remove(...i) {
```

Comment:
`remove` has only 1 argument. Why is are you using the [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) here?